AniRPG.MapSystem.Domain/Common/IEntity.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace AniRPG.MapSystem.Domain.Common
+{
+    public interface IEntity
```

Comment:
We dont need IEntity for specific MapSystem. Other systems could also use IEntity.
So we have 2 options:
1)create some global project with IEntity
2)remove IEntity interface for now

I suggest 2nd option for current version.

Reply:
IEntity deleted for now